git_hooks/pre-commit

```diff
@@ -1,39 +1,85 @@
 #!/bin/bash
 
-# run_lint: runs prettier and eslint on all staged files inside docker
-# files will be reformatted by prettier and the script will fail if there are
-# any warnings from eslint
-#  $1: container name
-#  $2: file names as in container
-#  $3: local file names
+shopt -s globstar nullglob
+
+# docker_exec:	Helper function
+docker_exec () {
+	local container="$1"
+	shift	# We don't need $1 any longer.
+
+	docker compose \
+		-f docker-compose.yml \
+		-f docker-compose-local.yml \
+                --env-file env-local.env \
```

Comment:
This command looks very similar to the one in `proj.sh` below, so I'm suspecting that it would be more maintainable to have have it in a single place: 
https://github.com/MetabolicAtlas/MetabolicAtlas/blob/86d69b91ae3f787fea3eeec86d67aecef53d76ce/proj.sh#L46

Reply:
Good point! Fixed in 28a886e